front_end/html/edit_assignment.html

```diff
@@ -23,6 +23,164 @@ <h2>Create Assignment</h2>
   <p><strong>Introduction: </strong><br />Please use <a href="https://www.markdownguide.org/basic-syntax/" target='_blank' rel='noopener noreferrer'>Markdown syntax</a> to write an introduction for this assignment.</p>
   <p><textarea name="introduction" class="textarea is-info is-normal is-medium is-fullwidth" placeholder="Please provide an introduction for this assignment." rows="8" style="font-family:monospace">{{ assignment_details["introduction"] }}</textarea></p>
 
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/flatpickr/dist/flatpickr.min.css">
+  <script src="https://cdn.jsdelivr.net/npm/flatpickr"></script>
+
+  <p><strong>Start date: </strong><br />If a start date is specified, students will be unable to view this assignment before the start date and time.</p>
+  <div class="select is-info">
+    <select name="has_start_date" onchange="showStartOptions(this.value)" style="padding: 2px;">
+      <option
+        {% if not assignment_details["start_date"] %}
+          selected
+        {% end %}
+      >None</option>
+      <option
+        {% if assignment_details["start_date"] %}
+          selected
+        {% end %}
+      >Select</option>
+    </select>
+  </div>
+  <div id="start_date"
+  {% if assignment_details["start_date"] %}
+    style="display: inline;"
+  {% else %}
+    style="display: none;"
+  {% end %}>
+    <input class="is-medium" id="start_date_picker" name="start_date_picker" type="text" placeholder="Select Date.." data-input>
```

Comment:
Please show me the look and feel of this.

Also, the placeholder should either have one period or three periods (ellipsis) but not two.